i18n/en/poll.csv

```diff
@@ -1,6 +1,7 @@
-poll.created, "Poll created successfully, here's the id for results tracking and other stuff ${pollId}"
+poll.created, "Poll created successfully, here's the ID for results tracking and other stuff: ${pollId}"
 poll.alert, "Hey everyone @here, a new poll just popped up! vote by typing /vote your_answer"
-poll.header, "${day}/${month} ${question} | ${votes} votes | ID${pollId}\n"
+poll.header, "${day}/${month} ${question} | ${votes} votes | ID: ${pollId}\n"
+poll.header.wstatus, "${day}/${month} ${question} | ${votes} votes | ID: ${pollId} (${status})\n"
```

Comment:
what doesnt mean "wstatus" here? what u want here? and why, one poll line named "header"... its like a "details"